### ISSUE TITLE
ImportError: cannot import name 'buf_t' from 'billiard.compat'

### ISSUE THREAD

**Issue by m8522s** [paperless-ngx/paperless-ngx#3896]
### Description

After update Paperless-NGX to version 1.16.5, the `paperless-task-queue` fails to start.

Python 3.9.2
celery 5.2.7 (dawn-chorus)
Redis server 6.0.16

### Steps to reproduce

`systemctl restart paperless-task-queue`

### Webserver logs

```bash
Jul 30 20:48:01 paperless systemd[1]: Started Paperless Celery Workers.
Jul 30 20:48:02 paperless celery[2386]: Traceback (most recent call last):
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/bin/celery", line 8, in <module>
Jul 30 20:48:02 paperless celery[2386]:     sys.exit(main())
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/celery/__main__.py", line 15, in main
Jul 30 20:48:02 paperless celery[2386]:     sys.exit(_main())
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/celery/bin/celery.py", line 217, in main
Jul 30 20:48:02 paperless celery[2386]:     return celery(auto_envvar_prefix="CELERY")
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 1130, in __call__
Jul 30 20:48:02 paperless celery[2386]:     return self.main(*args, **kwargs)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 1055, in main
Jul 30 20:48:02 paperless celery[2386]:     rv = self.invoke(ctx)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 1655, in invoke
Jul 30 20:48:02 paperless celery[2386]:     sub_ctx = cmd.make_context(cmd_name, args, parent=ctx)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 920, in make_context
Jul 30 20:48:02 paperless celery[2386]:     self.parse_args(ctx, args)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 1378, in parse_args
Jul 30 20:48:02 paperless celery[2386]:     value, args = param.handle_parse_result(ctx, opts, args)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 2360, in handle_parse_result
Jul 30 20:48:02 paperless celery[2386]:     value = self.process_value(ctx, value)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 2316, in process_value
Jul 30 20:48:02 paperless celery[2386]:     value = self.type_cast_value(ctx, value)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/click/core.py", line 2304, in type_cast_value
Jul 30 20:48:02 paperless celery[2386]:     return convert(value)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/click/types.py", line 82, in __call__
Jul 30 20:48:02 paperless celery[2386]:     return self.convert(value, param, ctx)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/celery/bin/worker.py", line 58, in convert
Jul 30 20:48:02 paperless celery[2386]:     value = concurrency.get_implementation(worker_pool)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/celery/concurrency/__init__.py", line 28, in get_implementation
Jul 30 20:48:02 paperless celery[2386]:     return symbol_by_name(cls, ALIASES)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/kombu/utils/imports.py", line 59, in symbol_by_name
Jul 30 20:48:02 paperless celery[2386]:     module = imp(module_name, package=package, **kwargs)
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
Jul 30 20:48:02 paperless celery[2386]:     return _bootstrap._gcd_import(name[level:], package, level)
Jul 30 20:48:02 paperless celery[2386]:   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
Jul 30 20:48:02 paperless celery[2386]:   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
Jul 30 20:48:02 paperless celery[2386]:   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
Jul 30 20:48:02 paperless celery[2386]:   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
Jul 30 20:48:02 paperless celery[2386]:   File "<frozen importlib._bootstrap_external>", line 790, in exec_module
Jul 30 20:48:02 paperless celery[2386]:   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/celery/concurrency/prefork.py", line 19, in <module>
Jul 30 20:48:02 paperless celery[2386]:     from .asynpool import AsynPool
Jul 30 20:48:02 paperless celery[2386]:   File "/usr/local/lib/python3.9/dist-packages/celery/concurrency/asynpool.py", line 29, in <module>
Jul 30 20:48:02 paperless celery[2386]:     from billiard.compat import buf_t, isblocking, setblocking
Jul 30 20:48:02 paperless celery[2386]: ImportError: cannot import name 'buf_t' from 'billiard.compat' (/usr/local/lib/python3.9/dist-packages/billiard/compat.py)
Jul 30 20:48:02 paperless systemd[1]: paperless-task-queue.service: Main process exited, code=exited, status=1/FAILURE
```

### Browser logs

```bash
None. The web site in the browser looks normal.
```

### Paperless-ngx version

1.16.5

### Host OS

Debian 11.7 amd64

### Installation method

Bare metal

### Browser

Firefox 115.0.3

### Configuration changes

Update from 1.14.5 to 1.16.5

### Other

-

**Comment by shamoon** [comment]
This very much seems like an issue with your installation, or at best celery, not paperless.

```
paperless celery[2386]: ImportError: cannot import name 'buf_t' from 'billiard.compat'
```

Closest I can find is https://github.com/celery/celery/issues/7583

Did you update dependencies when you upgraded? If youre managing a bare metal installation that responsibility is yours

**Comment by stumpylog** [comment]
Yeah, I agree there's a dependency issue with your bare metal install.  1.16 was released with [celery 5.3.0](https://github.com/paperless-ngx/paperless-ngx/blob/v1.16.5/Pipfile.lock#L188), not 5.2.7 as you listed.

**Comment by m8522s** [comment]
Thanks for your quick reply. I will update Celery to 5.3.0.
